### PR TITLE
Support for Launcher.QueryUriSupportAsync on Skia.Gtk and Skia.Wpf

### LINKED ISSUE #5770: Launcher.QueryUriSupportAsync for Skia.Gtk and Skia.Wpf
## What would you like to be added:
Add `Launcher.QueryUriSupportAsync` for Skia.Gtk and Skia.Wpf
## Why is this needed:
To check whether an `Uri` scheme can be launched, and for platform parity.
## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [x] Skia
  - [x] WPF
  - [x] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

## Anything else we need to know?
There are already many open source solutions for Windows and Linux, just need to port them to this platform?
Maybe [read the registry](https://github.com/trungnt2910/URIScheme/blob/3ccd79d15e92a7597bf231e37deeeb86922c99cd/URIScheme/WindowsURISchemeService.cs#L56) for WPF on Windows and [query xdg-settings and check for mimeapps.list](https://github.com/trungnt2910/URIScheme/blob/3ccd79d15e92a7597bf231e37deeeb86922c99cd/URIScheme/LinuxXdgURISchemeService.cs#L48) on [most] Linux distros?

### PR DESCRIPTION
**Pull request by trungnt2910**
GitHub Issue (If applicable): closes #5770

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the current behavior?
`Launcher.QueryUriSupportAsync` throws `NotImplementedException` on WPF and Gtk platforms.

## What is the new behavior?
`Launcher.QueryUriSupportAsync` tries to query for support by using the Registry (on Windows) and by using `xdg-settings` (on Linux).

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.Gtk/System/LauncherExtension.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			});
+		}
+
+		private bool CheckXdgSettings(Uri uri)
+		{
+			var process = new Process()
+			{
+				StartInfo = new ProcessStartInfo()
+				{
+					FileName = "xdg-settings",
```
Comment (jeromelaban) [inline src/Uno.UI.Runtime.Skia.Gtk/System/LauncherExtension.cs:47]: Is this available on all OSes ?

Comment (trungnt2910) [reply src/Uno.UI.Runtime.Skia.Gtk/System/LauncherExtension.cs:47]: It's part of `xdg-utils`, which should be available for all Linux distros with a desktop-environment.

Comment (trungnt2910) [reply src/Uno.UI.Runtime.Skia.Gtk/System/LauncherExtension.cs:47]: I might add a fallback by [reading the `mimeapps.list` file](https://github.com/trungnt2910/URIScheme/blob/master/URIScheme/Tools/MimeAppsList.cs), where URI schemes should be listed. This method should be available for most Linux Gtk desktops.

**Comment by trungnt2910** [comment]
Am I doing anything wrong? Why's the build cancelled?

File: src/Uno.UI.Runtime.Skia.Gtk/System/LauncherExtension.cs (hunk 1 above)
Comment (trungnt2910) [reply src/Uno.UI.Runtime.Skia.Gtk/System/LauncherExtension.cs:60]: Tested latest nuget package from artifacts on a Linux machine without `xdg-settings`, and the latest build works.

**Review by MartinZikmund — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@trungnt2910 Can you check the conflicts please? Hopefully can be merged for 3.8 👍

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**